content/api.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Commands Api"
+anchor: "commands-api"
+weight: 40
+---
+
+Meeseeks support commands to be invoked using a http API endpoint.
+
+This can be particularly useful when wired to an alerting system as this can be
+the seed to auto-remediation.
+
+Tokens can only be managed in a direct message conversation, because otherwise
+they would be leaking to any user who has access the used channel. Any attempt
+to invoke any token command in an open channel will be rejected by the
+meesseeks.
+
+## Creating an api token
+
+* `token-new @user #channel command [args...]`
+
+This invokation will then return a UUID which is the API token. This UUID will
+be linked to the command with the proposed arguments. The command will be
```

Comment:
Can the command be an alias?

Reply:
yes, because it will be lazily evaluated.

Reply:
This is why everything about commands happens inside commands, it's transparent for the rest of the layers.